plugins/vbe/fu-plugin-vbe.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * VBE plugin for fwupd,mmc-simple
+ *
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2022 Google LLC
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <fwupdplugin.h>
+
+enum { PROP_0, PROP_VBE_METHOD, PROP_VBE_FDT, PROP_VBE_NODE, PROP_LAST };
```

Comment:
What's this for?

Reply:
Ahh, so is the idea that all vbe devices will support these properties? The way we've done this before is make another superclass, i.e. `FuSimpleVbeDevice -> FuVbeDevice -> FuDevice -> FwupdDevice -> GObject` and then all the common properties can go in `FuVbeDevice` and all the special _simple_ logic can go in the top superclass. This is also why we allow the quick file to set the device GType -- i.e. you can say in the quirk file `create a FuSimpleVbeDevice for GUID XXXYYYYY` and the right things happens.

Reply:
Yes I expect these to be common properties. I was leaving that until we had another driver, but we don't need too. I will see if I can figure out what you have here.

Reply:
Done, but have not looked at quirks yet

Reply:
I'm going to leave quirks for the future.